incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,495 @@
+                .options(FileSystemOptionsConfig::builder)
+                    .keys()
+                        .store("") // env
+                        .type("") // env
+                        .password("") //env
+                        .build()
+                    .build()
+                .build();
+        }
+        return namespace;
+    }
+
+    private JsonPatch createEnvVarsPatch()
+    {
+        JsonArrayBuilder patch = Json.createArrayBuilder();
+        addOp(patch, "/bindings/tcp_client0/options/host", "${{env.TCP_CLIENT_HOST}}");
+        addOp(patch, "/bindings/tcp_client0/options/port", "${{env.TCP_CLIENT_PORT}}");
+
+        if (isJwtEnabled)
+        {
+            // jwt0 guard
+            addOp(patch, "/guards/jwt0/options/issuer", "${{env.JWT_ISSUER}}");
+            addOp(patch, "/guards/jwt0/options/audience", "${{env.JWT_AUDIENCE}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/alg", "${{env.JWT_ALG}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/kty", "${{env.JWT_KTY}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/kid", "${{env.JWT_KID}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/use", "${{env.JWT_USE}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/n", "${{env.JWT_N}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/e", "${{env.JWT_E}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/crv", "${{env.JWT_CRV}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/x", "${{env.JWT_X}}");
+            addOp(patch, "/guards/jwt0/options/keys/0/y", "${{env.JWT_Y}}");
+        }
+
+        if (isTlsEnabled)
+        {
+            // tls_server0 binding
+            addOp(patch, "/bindings/tls_server0/options/keys/0", "${{env.TLS_SERVER_KEYS}}");
+            addOp(patch, "/bindings/tls_server0/options/sni/0", "${{env.TLS_SERVER_SNI}}");
+            addOp(patch, "/bindings/tls_server0/options/alpn/0", "${{env.TLS_SERVER_ALPN}}");
+            // tls_client0 binding
+            addOp(patch, "/bindings/tls_client0/options/trust/0", "${{env.TLS_CLIENT_TRUST}}");
+            addOp(patch, "/bindings/tls_client0/options/sni/0", "${{env.TLS_CLIENT_SNI}}");
+            addOp(patch, "/bindings/tls_client0/options/alpn/0", "${{env.TLS_CLIENT_ALPN}}");
+            // client vault
+            addOp(patch, "/vaults/client/options/trust/store", "${{env.TRUSTSTORE_PATH}}");
+            addOp(patch, "/vaults/client/options/trust/type", "${{env.TRUSTSTORE_TYPE}}");
+            addOp(patch, "/vaults/client/options/trust/password", "${{env.TRUSTSTORE_PASSWORD}}");
+            // server vault
+            addOp(patch, "/vaults/server/options/keys/store", "${{env.KEYSTORE_PATH}}");
+            addOp(patch, "/vaults/server/options/keys/type", "${{env.KEYSTORE_TYPE}}");
+            addOp(patch, "/vaults/server/options/keys/password", "${{env.KEYSTORE_PASSWORD}}");
+        }
+
+        return JsonProvider.provider().createPatch(patch.build());
```

Comment:
Please use code similar to the following, instead of using an intermediate `JsonArrayBuilder` to build the raw patch document.

```
JsonPatchBuilder patch = Json.createPatchBuilder();

patch.replace("/bindings/tcp_client0/options/host", "${{env.TCP_CLIENT_HOST}}");
patch.replace("/bindings/tcp_client0/options/port", "${{env.TCP_CLIENT_PORT}}");

if (isJwtEnabled)
{
    ...
}

if (isTlsEnabled)
{
    ...
}

return patch.build();
```

Reply:
oops, looks much better, thx 😅